package.json

```diff
@@ -1,167 +1,169 @@
-{
```

Comment:
In Webstorm, go in "Preferences", then "Editor > Code Style" and change the line separator to use the Windows one. This should fix the EOL problem.